src/library.js

```diff
@@ -1328,6 +1351,52 @@ const createPilingJs = (rootElement, initOptions = {}) => {
       stateUpdates.add('layout');
     }
 
+    if (state.itemOpacity !== newState.itemOpacity) {
+      if (newState.itemOpacity) {
+        pileInstances.forEach(pile => {
+          pile.itemContainer.children.forEach((item, i) => {
```

Comment:
I think we should create a new update function called `updatePileItemStyle` and move the loop into there. Additionally, in `updatePileItem`, we should then also call `updatePileItemStyle`.